include/irr/asset/filters/CBlitImageFilter.h

```diff
@@ -192,17 +197,15 @@ class CBlitImageFilter : public CImageFilter<CBlitImageFilter<KernelX,KernelX,Ke
 				KernelX					kernelX;
 				KernelY					kernelY;
 				KernelZ					kernelZ;
-				asset::DefaultSwizzle defaultSwizzle;
 				bool normalize = false;
```

Comment:
compile time template parameter would be better

Reply:
everything is now tempalted

Reply:
oh u mean `normalize`

Reply:
so what kind of classes should I pass to `normalize`? or should it be only a bool?

Reply:
a it would doing the same like SAT

Reply:
> 
> 
> so what kind of classes should I pass to `normalize`? or should it be only a bool in template param?

SAT, ConvertSwizzle and Blit

do we have any more questions after discord chat?

Reply:
no, there are not

Reply:
now we've got an interface for that

Reply:
corrected